utils/notifications.js

```diff
@@ -705,7 +705,7 @@ const sendEmailLink = async (req, res) => {
         const body = {
             message: {
                 subject:
-                    preferredLanguage ===
+                    preferredLanguage.toString() ===
```

Comment:
No need to convert to strings if they both are numbers.

Reply:
Solved